plugins/prebuffer-mixin/src/stream-settings.ts

```diff
@@ -1,5 +1,5 @@
 import { getH264DecoderArgs } from "@scrypted/common/src/ffmpeg-hardware-acceleration";
-import { MixinDeviceBase, ResponseMediaStreamOptions, VideoCamera } from "@scrypted/sdk";
```

Comment:
you can remove all the changes from rebroadcast completely. this looks leftover.

Reply:
nevermind